costs/src/index.ts

```diff
@@ -3,37 +3,32 @@
  * DO NOT EDIT THIS FILE UNLESS IT IS IN /costs/src/index.ts
  */
 
-// TODO this can all move to Valhalla in Arch-ReDesign V2
+import { ModelRow } from "./interfaces/Cost";
+import { allCosts, defaultProvider, providers } from "./providers/mappings";
 
-import { costs as openaiCosts } from "./providers/openai";
-import { costs as fineTunedOpenAICosts } from "./providers/openai/fine-tuned-models";
-import { costs as togetherAIChatCosts } from "./providers/togetherai/chat";
-import { costs as togetherAIChatLlamaCosts } from "./providers/togetherai/chat/llama";
-import { costs as togetherAICompletionCosts } from "./providers/togetherai/completion";
-import { costs as togetherAICompletionLlamaCosts } from "./providers/togetherai/completion";
-import { costs as groqCosts } from "./providers/groq";
-import { costs as azureCosts } from "./providers/azure";
-import { costs as googleCosts } from "./providers/google";
-import { costs as anthropicCosts } from "./providers/anthropic";
+export function costOf({
+  model,
+  provider,
+}: {
+  model: string;
+  provider: string;
+}) {
+  const modelLower = model.toLowerCase();
+  const providerCost = providers.find((p) => {
```

Comment:
case insensitive

Reply:
that's fine, because provider is a text that is determined by us. We have more control over it